### ISSUE TITLE
Route [password.reset] not defined

### ISSUE THREAD

**Issue by keizah7** [laravel/nova-issues#3644]
- Laravel Version: 8.72
- Nova Version: 3.30
- PHP Version: 8.0.8
- Database Driver & Version:

Fresh install shows exception on resetting password.

```
Symfony\Component\Routing\Exception\RouteNotFoundException
Route [password.reset] not defined.
```
https://subdomain.domain.test/password/email 

Maybe it is related to #3542

**Comment by crynobone** [comment]
`password.reset` should be reserved for your application's main password reset page and Nova only uses `nova.password.reset`. Please use `php artisan route:list --name=nova` to check available route name registered by Nova.

**Comment by crynobone** [comment]
Broken via https://github.com/laravel/framework/pull/39652

**Comment by keizah7** [comment]
Route is registered 
![image](https://user-images.githubusercontent.com/49227202/142431399-71146d58-4066-47c3-8041-db74963e5aef.png)


Code below solved issue
```
<?php

namespace App\Providers;

use Illuminate\Auth\Notifications\ResetPassword;
use Illuminate\Support\ServiceProvider;

class AppServiceProvider extends ServiceProvider
{
    /**
     * Register any application services.
     *
     * @return void
     */
    public function register()
    {
        //
    }

    /**
     * Bootstrap any application services.
     *
     * @return void
     */
    public function boot()
    {
        ResetPassword::createUrlUsing(function ($user, string $token) {
            return route('nova.password.reset', $token);
        });
    }
}

```

**Comment by crynobone** [comment]
Fixed in 8.73.0
